2-mandatory/1-create-functions.js

```diff
@@ -44,8 +48,17 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
-}
+function formatPercentage(arr) {
+  arr.map( (number => {
```

Comment:
When you ran this code, what happened?  It doesn't look to me like anything can be returned from this function just now. 

Can you step through it again? Paste the requirements into the body of the function and break it down as simply as you can.

Reply:
thank you Raj has gone over this with me and I have ammended